src/main/java/hudson/plugins/jira/JiraSite.java

```diff
@@ -491,13 +484,7 @@ public String getReleaseNotesForFixVersion(String projectKey, String versionName
             String key = issue.getKey();
             String summary = issue.getSummary();
             String status = issue.getStatus().getName();
-            String type = "UNKNOWN";
```

Comment:
This doesn't look like it fixes the issue. Moreover it looks like it will not fallback to UNKNOWN in case the issueType is not known...
Also, this needs a test.